# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-contain/parsing/contain-valid.html.ini
# Commit: e25543fae7ee
# Full Hash: e25543fae7ee03ac57b9cfbfeb2418c9e2f490fc
# Author: Martin Robinson <mrobinson@igalia.com>
# Date: 2022-06-23 03:54:50
# Regressor Bug: 1463600
# File Overlap Count: 1
# Description:
#   Bug 1463600 - Implement CSS 'contain: style' r=emilio
#   
#   Add an implementation of CSS `contain: style`. This introduces two new
#   data structures, the ContainStyleScope and ContainStyleScopeManager.
#   
# ==============================================================================

diff -r 4230bdcf07ae -r e25543fae7ee testing/web-platform/meta/css/css-contain/parsing/contain-valid.html.ini
--- a/testing/web-platform/meta/css/css-contain/parsing/contain-valid.html.ini	Wed Jun 22 16:15:57 2022 +0000
+++ b/testing/web-platform/meta/css/css-contain/parsing/contain-valid.html.ini	Wed Jun 22 16:16:59 2022 +0000
@@ -1,15 +1,3 @@
 [contain-valid.html]
-  [e.style['contain'\] = "style" should set the property value]
-    expected: FAIL
-
-  [e.style['contain'\] = "paint style" should set the property value]
-    expected: FAIL
-
-  [e.style['contain'\] = "layout style paint" should set the property value]
-    expected: FAIL
-
-  [e.style['contain'\] = "layout paint style size" should set the property value]
-    expected: FAIL
-
   [e.style['contain'\] = "layout inline-size" should set the property value]
     expected: FAIL